ui/page/transaction/transaction_details_page.go

```diff
@@ -898,48 +874,26 @@ func (pg *TxDetailsPage) showbrowserURLModal(copyredirect *cryptomaterial.Clicka
 			return components.BrowserURLWidget(gtx, pg.Load, redirectURL, copyredirect)
 		}).
 		SetPositiveButtonText(values.String(values.StrGotIt))
-
-	pg.moreOptionIsOpen = false
 	pg.ParentWindow().ShowModal(info)
 }
 
-func (pg *TxDetailsPage) layoutOptionsMenu(gtx C) {
-	inset := layout.Inset{
-		Left: values.MarginPaddingMinus145,
-	}
-
-	m := op.Record(gtx.Ops)
-	inset.Layout(gtx, func(gtx C) D {
-		gtx.Constraints.Max.X = gtx.Dp(values.MarginPadding168)
-		return pg.shadowBox.Layout(gtx, func(gtx C) D {
-			optionsMenuCard := cryptomaterial.Card{Color: pg.Theme.Color.Surface}
-			optionsMenuCard.Radius = cryptomaterial.Radius(5)
-			return optionsMenuCard.Layout(gtx, func(gtx C) D {
-				return (&layout.List{Axis: layout.Vertical}).Layout(gtx, len(pg.moreItems), func(gtx C, i int) D {
-					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-						layout.Rigid(func(gtx C) D {
-							return pg.moreItems[i].button.Layout(gtx, func(gtx C) D {
-								return layout.UniformInset(values.MarginPadding10).Layout(gtx, func(gtx C) D {
-									if pg.moreItems[i].button.Clicked(gtx) {
-										switch pg.moreItems[i].id {
-										case viewBlockID: // redirect to browser
-											pg.showbrowserURLModal(pg.moreItems[i].button)
-											pg.moreOptionIsOpen = false
-										default:
-										}
-									}
-
-									gtx.Constraints.Min.X = gtx.Constraints.Max.X
-									return pg.Theme.Label(values.TextSize14, pg.moreItems[i].text).Layout(gtx)
-								})
-							})
-						}),
-					)
-				})
-			})
```

Comment:
why did you remove the option menu?

what happens on mobile?

Reply:
please checkout above conversation

Reply:
you didn't link any conversation